asm-multiproject-patch/resources/istio-operator.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Google LLC
```

Comment:
I'm still not clear how will the customer use citadel with multiproject with this PR? This istio-operator.yaml doesn't include anything related to Citadel and also you don't have another directory `asm-citadel-multiproject-patch`?

Maybe the PR is not updated with latest changes yet?

Reply:
The customer will actually use `asm-citadel-patch` to install multiproject, since we can completely reuse the same package.

Reply:
how? the `asm-citadel-patch` is quite different from the asm-multiproject-patch.

And also how does the customer use citadel with ` asm-multiproject`? Can you list the commands the user needed to run?

Reply:
For the current change in this PR. The customers need to use `asm-multiproject` pkg and `asm-multiproject-patch`pkg for mesh ca installation, and use `asm-citadel` pkg and `asm-citadel-patch` pkg for citadel installation. 

We can reuse `asm-citadel` and `asm-citadel-patch`.